src/huggingface_hub/templates/modelcard_template.md

```diff
@@ -81,15 +81,21 @@ Use the code below to get started with the model.
 
 {{ training_data | default("[More Information Needed]", true)}}
 
-## Training Procedure [optional]
+## Training Procedure 
 
 <!-- This relates heavily to the Technical Specifications. Content here should link to that section when it is relevant to the training procedure. -->
 
-### Preprocessing
+### Preprocessing [optional]
 
 {{ preprocessing | default("[More Information Needed]", true)}}
 
-### Speeds, Sizes, Times
+
+### Training Hyperparameters
+
+- **Training regime:** {{ training_regime | default("[More Information Needed]", true)}} <!--fp32, fp16 mixed precision, bf16 mixed precision, bf16 non-mixed precision, fp16 non-mixed precision, fp8 mixed precision -->
+- **Regime:** [More Information Needed] <!--mixed precision or not -->
```

Comment:
```suggestion
```

This line is not needed anymore since we now have the combined `Training regime:` section, right?